ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/PullQueryExecutorMetrics.java

```diff
@@ -46,19 +48,19 @@
   private final Sensor latencySensor;
   private final Sensor requestRateSensor;
   private final Sensor errorRateSensor;
+  private final Sensor requestSizeSensor;
+  private final Sensor responseSizeSensor;
   private final Metrics metrics;
   private final Map<String, String> customMetricsTags;
-  private final Time time;
   private final String ksqlServiceId;
+  private final Time time = Time.SYSTEM;
 
   public PullQueryExecutorMetrics(
       final String ksqlServiceId,
-      final Map<String, String> customMetricsTags,
-      final Time time
+      final Map<String, String> customMetricsTags
   ) {
 
     this.customMetricsTags = Objects.requireNonNull(customMetricsTags, "customMetricsTags");
-    this.time = Objects.requireNonNull(time, "time");
```

Comment:
If you leave time injected in, it'll make it easier to test this class.  In fact, I would imagine it would be hard to test if it's using a wall clock.